### DISCUSSION TITLE
Error creating extended property

### DISCUSSION THREAD

**Discussion by Songjiann** [dbeaver/dbeaver#36640] [DBeaver Development]
<img width="923" alt="1735177988310" src="https://github.com/user-attachments/assets/a0cb2992-3a44-4a1d-ad8e-f5bbde373935" />
<img width="723" alt="1735178069501" src="https://github.com/user-attachments/assets/047516b1-b305-4f2e-8395-f4da3cc64fa4" />
Error on creating extended properties when table page is not open. How should I change it?

**Comment by serge-rider** [comment]
It looks like a bug in SQL Server extension. Could you create a ticket at https://github.com/dbeaver/dbeaver/issues ?

Workaround: create a table first, save it with ctrl+s and then add an extended property

**Reply by Songjiann** [reply]
Thank you for your answer ! Here's how I solved it
<img width="835" alt="1736304579997" src="https://github.com/user-attachments/assets/1e6aa0cf-3c4c-411e-b23a-acf497c4f8dc" />

**Reply by Songjiann** [reply]
I submitted the bug like you said 
<img width="920" alt="1736305501809" src="https://github.com/user-attachments/assets/c8c8dcf3-1c41-4dd4-a0a4-d00ccd835692" />
